src/components/edit/earthEngine/BandSelect.js

```diff
@@ -5,16 +5,29 @@ import { connect } from 'react-redux';
 import { SelectField } from '../../core';
 import { setBand } from '../../../actions/layerEdit';
 
-const BandSelect = ({ band = [], bands, setBand, errorText }) => (
-    <SelectField
-        label={i18n.t('Groups')}
-        items={bands}
-        multiple={true}
-        value={band}
-        onChange={setBand}
-        errorText={!band.length && errorText ? errorText : null}
-    />
-);
+const MAX_BANDS = 10;
+
+const BandSelect = ({ band = [], bands, setBand, errorText }) => {
+    let warning;
+
+    if (band.length > MAX_BANDS) {
+        warning = i18n.t(
+            'Warning: It takes longer time to calculate data for many groups.'
```

Comment:
Does it also consume a lot of data to choose a lot of groups? If so, we could say:
`Choosing many groups downloads a lot of data and takes time to display.`
If it's just calculation time, then it could be:
`Choosing many groups takes a long time to calculate and display.`

Reply:
Thanks @cooper-joe! It's just calculation time so used the latter.